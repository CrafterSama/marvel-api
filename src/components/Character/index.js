import React from 'react';
import { Container } from '../../styles';
import SearchResults from '../Common/SearchResults';

const Character = () => {

  return (
    <Container>
      <SearchResults />
    </Container>
  );
};

export default Character;
