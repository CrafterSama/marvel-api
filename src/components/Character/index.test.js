import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';
import Character from './index';

configure({ adapter: new Adapter() });

describe('Test for <Character /> Component', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<Character />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
