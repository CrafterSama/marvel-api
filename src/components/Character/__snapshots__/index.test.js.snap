// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <Character /> Component matches snapshot 1`] = `
<styles__Container>
  <SearchResults />
</styles__Container>
`;
